cmake/test-targets.cmake

```diff
@@ -37,6 +37,17 @@ add_test(
     WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
 )
 
+find_program(SHELLCHECK_PATH shellcheck)
+if (SHELLCHECK_PATH)
+    add_test(
+        NAME test-local-shellcheck
+        COMMAND shellcheck -x $(file --mime-type script/* t/* container/worker/*.sh tools/* | sed -n 's/^\(.*\):.*text\/x-shellscript.*$/\\1/p')
```

Comment:
SHELLCHECK_PATH should be used there.

Reply:
I needed to change that now similar as we do for the python and yaml style check by passing the SHELLCHECK_PATH as first argument to the check script